.github/workflows/changelog.yaml

```diff
@@ -0,0 +1,43 @@
+name: Changelog
```

Comment:
```suggestion
name: Validate Changelog
```

Reply:
I will add another task in this file to auto create a changelog for PR, so I think `Changelog` is good to use